JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -92,12 +88,25 @@ module Microsoft.ApplicationInsights {
             };
         }
 
+        private openHandler(xhr: XMLHttpRequestInstrumented, method, url, async) {
+            var ajaxData = new ajaxRecord();
+            ajaxData.method = method;
+            ajaxData.requestUrl = url;
+            ajaxData.requestSize += url.length;
+            // If not set async defaults to true 
+            ajaxData.async = extensions.IsNullOrUndefined(async) ? true : async;
```

Comment:
As I mentioned we are making work to remove async attribute from schema. Also it's nullable - you should not default it to true.